tests/simple/testdata/integer_math.textproto

```diff
@@ -200,8 +198,14 @@ section {
     eval_error: {
       errors: { message: "return error for overflow" }
     }
-    ## Current behavior:
-    ## value: { int64_value: 9223372036854775807 }
+  }
+  test {
+    name: "int64_min_negate"
+    description: "Negated LLONG_MIN is not representable."
```

Comment:
nit: extra `L` in `LONG`?

Reply:
I think Gabriel was using standard C constants, where "long" might only be 32-bits, so it's for "long long min", guaranteeing at least 64 bits. Long long could be bigger than that, but C's flexibility for integer sizes is mostly theoretical these days - at least outside the embedded market.